www/pages/plugins/source.md

```diff
@@ -0,0 +1,64 @@
+---
+label: 'source'
+menu: side
+title: 'Source'
+index: 6
+---
+
+## Source
+
+The source plugin allows users to include external content as pages that will be statically generated just like if they were a markdown or HTML in your _pages/_ directory.  This would be the primary API to include content from a headless CMS, database, the filesystem, SaaS provider (Notion, AirTables) or wherever else you keep it.
+
+> See our [docs on _External Sources_](/docs/data/#external-sources) for more information on working with data in Greenwood.
+
+## API
+This plugin supports providing an array of "page" objects that will be added as nodes in [the graph](/docs/data/).
+
+```js
+// my-source-plugin.js
+export const customExternalSourcesPlugin = (options = {}) => {
+  type: 'source',
+  name: 'source-plugin-myapi',
+  provider: () => {
+    return async function () {
+      // this could just as easily come from an API, DB, Headless CMS, etc
+      const artists = await fetch('http://www.myapi.com/...').then(resp => resp.json());
```

Comment:
Not sure if this a good example, or maybe it would be best to keep it simple and just use a blog as the reference?